h/views/main.py

```diff
@@ -87,7 +87,10 @@ def _html_link(request, annotation):
 
 def _incontext_link(request, annotation):
     """Generate a link to an annotation on the page where it was made."""
-    if not request.feature('direct_linking'):
+
+    # Direct links are currently not supported for replies
+    is_reply = bool(annotation.get('references'))
```

Comment:
I'm not sure that this is a very Pythonic way to achieve this. We also have a similar `is_reply` helper in one other place in the code. Anyone have suggestions on a better way to do this?